### ISSUE TITLE
ClassCastException during querying provider ClassLineMarkerProvider

### ISSUE THREAD

**Issue by kaleev** [AxonFramework/IdeaPlugin#59]
IntelliJ IDEA build 213.7172.25

During querying provider org.axonframework.intellij.ide.plugin.markers.ClassLineMarkerProvider@e9cc35d (class org.axonframework.intellij.ide.plugin.markers.ClassLineMarkerProvider)

```
java.lang.ClassCastException: class com.intellij.psi.PsiWildcardType cannot be cast to class com.intellij.psi.PsiClassType (com.intellij.psi.PsiWildcardType and com.intellij.psi.PsiClassType are in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @2a0934a4)
	at org.axonframework.intellij.ide.plugin.util.PSiProcessingUtilsKt.toQualifiedName(PSiProcessingUtils.kt:56)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.isOfType(AggregateStructureResolver.kt:116)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.isMap(AggregateStructureResolver.kt:112)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.isCollection(AggregateStructureResolver.kt:111)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.inspect(AggregateStructureResolver.kt:80)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.access$inspect(AggregateStructureResolver.kt:40)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver$resolve$1.invoke(AggregateStructureResolver.kt:70)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver$resolve$1.invoke(AggregateStructureResolver.kt:67)
	at org.axonframework.intellij.ide.plugin.util.PerformanceRegistry.measure(PerformanceRegistry.kt:53)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.resolve(AggregateStructureResolver.kt:67)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.access$resolve(AggregateStructureResolver.kt:40)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver$cache$1.invoke(AggregateStructureResolver.kt:41)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver$cache$1.invoke(AggregateStructureResolver.kt:41)
	at org.axonframework.intellij.ide.plugin.util.CacheUtilsKt.createCachedValue$lambda-0(CacheUtils.kt:48)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:39)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:227)
	at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:42)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:227)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:114)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:44)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:68)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:228)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:28)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.getModels(AggregateStructureResolver.kt:43)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.getFlattendedModelsAndEntities(AggregateStructureResolver.kt:45)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.getMemberForName(AggregateStructureResolver.kt:57)
	at org.axonframework.intellij.ide.plugin.resolving.AggregateStructureResolver.getAllModelsRelatedToName(AggregateStructureResolver.kt:48)
	at org.axonframework.intellij.ide.plugin.markers.ClassLineMarkerProvider.getLineMarkerInfo(ClassLineMarkerProvider.kt:62)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.queryProviders(LineMarkersPass.java:165)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.lambda$doCollectInformation$3(LineMarkersPass.java:98)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:115)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:85)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:56)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:414)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1084)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:407)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:624)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:698)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:646)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:623)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:406)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:382)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:181)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:380)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:184)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by Morlack** [comment]
Hello @kaleev, thanks for reporting the bug to us. I have found the issue and will release the fix with `0.6.2`

**Comment by Morlack** [comment]
I have submitted release `0.6.2` containing this fix to Jetbrains. It should be processed within 2 days and become available as update
